service/src/main/java/uk/gov/hmcts/reform/fpl/service/LocalAuthorityValidationService.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class LocalAuthorityValidationService {
+    private final OrganisationService organisationService;
+
+    public List<String> validateIfLaIsOnboarded(final String localAuthorityCode, String currentUser) {
+        List<String> errors = new ArrayList<>();
+
+        Set<String> users = organisationService.findUserIdsInSameOrganisation(localAuthorityCode);
```

Comment:
probably findOrganisationById() is better option here. I dont think that findUserIdsInSameOrganisation can be used, as it should still have fallback part that check out conf